include/fi_enosys.h

```diff
@@ -150,7 +150,7 @@ int fi_no_atomic_compwritevalid(struct fid_ep *ep,
 /*
 static struct fi_ops_cm X = {
 	.size = sizeof(struct fi_ops_cm),
-	.getname = X,
+	.getname = fi_no_getname,
```

Comment:
My bad. The fi_no_getname function already existed so I assumed that the comment hadn't been updated. I retract my statement about one of the providers not implementing .getname, I was mistaken.